media/js/firefox/new.js

```diff
@@ -27,6 +62,21 @@
                 // (from a download button) then we want them to see the same scene 2
                 // as non-firefox users and initiate a download
                 $html.addClass('firefox-latest');
+
+                uiTourGetConfiguration('appinfo', function(config) {
```

Comment:
Since the markup for this change is now wrapped in an l10n tag, is it worth adding a check that the markup exists before querying `appinfo`? The callback won't error with what's currently happening, but could still be a worthwhile safety check maybe?